nixos/modules/config/zram.nix

```diff
@@ -177,9 +179,9 @@ in
           serviceConfig = {
             Type = "oneshot";
             RemainAfterExit = true;
-            ExecStartPre = "${modprobe} -r zram";
-            ExecStart = "${modprobe} zram";
-            ExecStop = "${modprobe} -r zram";
+            ExecStartPre = "-${modprobe} -r zram";
+            ExecStart = "-${modprobe} zram";
```

Comment:
```suggestion
            ExecStart = "-${modprobe} zram num_devices=${toString cfg.numDevices}";
```

Reply:
This is already covered by the modprobe config set above.

Reply:
I can't find it where it is defined. I only see the `boot.kernelParams` but I don't think it would apply without rebooting.

Reply:
https://github.com/NixOS/nixpkgs/blob/be07c1bad9f639b1e739eaa94605d939efe759b5/nixos/modules/config/zram.nix#L130-L133